native/cocos/bindings/auto/jsb_gfx_auto.cpp

```diff
@@ -21418,6 +21418,9 @@ SE_BIND_FUNC(js_gfx_Device_createShader)
 
 static bool js_gfx_Device_createSwapchain(se::State& s) // NOLINT(readability-identifier-naming)
 {
+	// linwei: multi-windows hack
+    return true;
+
```

Comment:
native/cocos/bindings/auto/XXX is auto generated. Modifications might be overwritten.

Reply:
You're right，just a testing code.